### PR TITLE
bring back slot time check

### PR DESCRIPTION
**Pull request by kziemianek**
Brings back slot time check only in case there are more than one worker registered in teerex registry.

### CODE REVIEW

File: tee-worker/enclave-runtime/src/top_pool_execution.rs [hunk 1]
```diff
@@ -271,8 +271,17 @@ where
 		)
 		.with_claim_strategy(SlotClaimStrategy::RoundRobin);
 
+	let is_single_worker = match ocall_api.get_trusted_peers_urls() {
```
Comment (kziemianek) [inline tee-worker/enclave-runtime/src/top_pool_execution.rs:317]: We only check if there are more workers registered, which might not really mean they are online and syncing sidechain state but that should be enough for now.

File: tee-worker/sidechain/consensus/slots/src/lib.rs [hunk 2]
```diff
@@ -558,6 +556,7 @@ impl<ParentchainBlock: ParentchainBlockTrait, T: SimpleSlotWorker<ParentchainBlo
 		&mut self,
 		slot_info: SlotInfo<ParentchainBlock>,
 		shards: Vec<Self::ShardIdentifier>,
+		single_worker: bool,
```
Comment (Kailai-Wang) [inline tee-worker/sidechain/consensus/slots/src/lib.rs]: Could you make this variable consistent with previously declared one? (i.e. `is_single_worker`)

File: tee-worker/enclave-runtime/src/top_pool_execution.rs (hunk 1 above)
Comment (Kailai-Wang) [reply tee-worker/enclave-runtime/src/top_pool_execution.rs:317]: Could you add this into code comment? :)

**Review by Kailai-Wang — APPROVED** [review]
Thanks, I like the unit tests

**Pull request merged**